cmake/FindMASA.cmake

```diff
@@ -0,0 +1,35 @@
+# Find MASA library
+#
+# Set MASA_DIR to the base directory where the package is installed
+#
+# Sets 4 variables
+#   - MASA_INCLUDE_DIRS
+#   - MASA_MOD_DIRS
+#   - MASA_LIBRARY
+#   - MASA_FORTRAN_LIBRARY
+#
+
+find_path(MASA_INCLUDE_DIRS
+  masa.h
+  HINTS ${MASA_DIR} ${CMAKE_INSTALL_PREFIX}
+  PATH_SUFFIXES include)
+
+find_path(MASA_MOD_DIRS
+  masa.mod
+  HINTS ${MASA_DIR} ${CMAKE_INSTALL_PREFIX}
+  PATH_SUFFIXES lib)
+
+find_library(MASA_LIBRARY
+  NAMES masa
+  HINTS ${MASA_DIR} ${CMAKE_INSTALL_PREFIX}
+  PATH_SUFFIXES lib)
+
+find_library(MASA_FORTRAN_LIBRARY
```

Comment:
Is this needed  for our use case? Can MASA work as a pure C/C++ library? If so, we shouldn't error out if Fortran is not available.